src/backend/metal/src/device.rs

```diff
@@ -179,7 +179,7 @@ impl PhysicalDevice {
                 argument_buffers: Self::supports_any(device, ARGUMENT_BUFFER_SUPPORT) && false, //TODO
                 shared_textures: !Self::is_mac(device),
                 format_depth24_stencil8: device.d24_s8_supported(),
-                format_depth32_stencil8: false, //TODO: crashing the Metal validation layer upon copying from buffer
+                format_depth32_stencil8: true, //TODO: crashing the Metal validation layer upon copying from buffer
```

Comment:
So this works now? 😃

Reply:
not really, but Dota uses that, and we should just file an actual Apple bugreport if this doesn't work instead of avoiding it